bigframes/core/__init__.py

```diff
@@ -234,6 +239,10 @@ def _ibis_order(self) -> Sequence[ibis_types.Value]:
             self._ordering.all_ordering_columns,
         )
 
+    @property
+    def api_method(self) -> Dict[str, str]:
```

Comment:
Maybe a list of named tuples would be a good way to represent this?

Reply:
I've considered it before, and I think it is a data structure choice. 1. `job_configs.labels` is a dictionary, so we do less work assigning variable there. 2. In python, we can sort the dictionary by key and value, so I don't think there is much difference than using a list of tuples for doing any operations.